js/src/setup-mc/setup-stepper/setup-paid-ads/setup-paid-ads.js

```diff
@@ -24,37 +27,68 @@ import PaidAdsFeaturesSection from './paid-ads-features-section';
 import GoogleAdsAccountSection from './google-ads-account-section';
 import AudienceSection from '.~/components/paid-ads/audience-section';
 import BudgetSection from '.~/components/paid-ads/budget-section';
+import BillingCard from '.~/components/paid-ads/billing-card';
 import validateForm from '.~/utils/paid-ads/validateForm';
 import { getProductFeedUrl } from '.~/utils/urls';
-import { GUIDE_NAMES, GOOGLE_ADS_ACCOUNT_STATUS } from '.~/constants';
 import { API_NAMESPACE } from '.~/data/constants';
+import {
+	GUIDE_NAMES,
+	GOOGLE_ADS_ACCOUNT_STATUS,
+	GOOGLE_ADS_BILLING_STATUS,
+} from '.~/constants';
 
-const { CONNECTED } = GOOGLE_ADS_ACCOUNT_STATUS;
-
-function PaidAdsSectionsGroup( { onCampaignChange } ) {
+function PaidAdsSectionsGroup( { onStatesReceived } ) {
 	const { googleAdsAccount } = useGoogleAdsAccount();
 	const { data: targetAudience } = useTargetAudienceFinalCountryCodes();
+	const { billingStatus } = useGoogleAdsAccountBillingStatus();
 
-	if ( ! targetAudience ) {
-		return <GoogleAdsAccountSection />;
-	}
+	const onStatesReceivedRef = useRef();
+	onStatesReceivedRef.current = onStatesReceived;
 
-	const handleChange = ( _, values, isValid ) => {
-		onCampaignChange( { ...values, isValid } );
+	const initialValues = {
+		amount: 0,
+		countryCodes: targetAudience,
 	};
+	const [ campaign, setCampaign ] = useState( {
+		...initialValues,
+		isValid: ! Object.keys( validateForm( initialValues ) ).length,
+		isReady: false,
+	} );
```

Comment:
📝 
The initial values processing here aren't applied to Form `initialValues` below is temporary and will be changed in a separate PR, which implements the states storing and restoring for the Form.

https://github.com/woocommerce/google-listings-and-ads/blob/174b70a56d9e819f71c4a52c34c12baddd83a533/js/src/setup-mc/setup-stepper/setup-paid-ads/setup-paid-ads.js#L77-L80